src/main/java/net/fabricmc/loader/transformer/accessWidener/AccessWidener.java

```diff
@@ -195,21 +195,46 @@ public Access getMethodAccess(EntryTriple entryTriple) {
 		return classes;
 	}
 
+	private static int makePublic(int i) {
+		return (i & ~(Opcodes.ACC_PRIVATE | Opcodes.ACC_PROTECTED)) | Opcodes.ACC_PUBLIC;
+	}
+
+	private static int makeProtected(int i) {
+		if ((i & Opcodes.ACC_PRIVATE) != 0) {
+			//Return i if public
+			return i;
+		}
```

Comment:
The method could still have package visibility.

Reply:
I think ive fixed this now.